.github/workflows/hub-api-cd.yml

```diff
@@ -22,6 +24,7 @@ env:
   HUB_LIST_LAMBDA: hubapi_list
```

Comment:
Need to add `JINA_DOCKER_USERNAME` & `JINA_DOCKER_PASSWORD` env variables here (to read from secrets)